tests/test_tool.rs

```diff
@@ -0,0 +1,149 @@
+use rocksdb::run_ldb_cmd;
+use tempdir::TempDir;
+
+#[inline]
+fn generate_vec_string(s: &String) -> Vec<String> {
+    let d: Vec<_> = s
+        .split(' ')
+        .collect::<Vec<_>>()
+        .iter()
+        .map(|s| s.to_string())
+        .collect();
```

Comment:
The first `collect::<Vec<_>>()` is redundant, just use 

```rust
s.split(' ').map(|s| s.to_string()).collect()
```

Reply:
done.